lib/bunny/transport.rb

```diff
@@ -349,8 +353,8 @@ def prepare_tls_context(opts)
       @tls_key               = tls_key_from(opts)
       @tls_certificate_store = opts[:tls_certificate_store]
 
-      @tls_ca_certificates   = opts.fetch(:tls_ca_certificates, default_tls_certificates)
-      @verify_peer           = (opts[:verify_ssl] || opts[:verify_peer])
+      @tls_ca_certificates   = tls_ca_certificates_paths_from(opts) || default_tls_certificates
```

Comment:
I susptect that `tls_ca_certificates_paths_from` will never evaluate to nil, so the `default_tls_certificates` will never be used

Reply:
You are totally right, what do you think if we call `Array()` out of `#tls_ca_certificates_paths_from` method?
```
@tls_ca_certificates   = Array(tls_ca_certificates_paths_from(opts) || default_tls_certificates)
```
Should I create a PR with fix?

Reply:
yes, great